docs/tutorial/tutorial-1.rst

```diff
@@ -52,6 +52,22 @@ The second argument (``relativeToURL``) is accessed as a keyword argument. This
 argument is declared as being of type ``NSURL *``; since ``base`` is an
 instance of ``NSURL``, Rubicon can pass through this instance.
 
+If the same argument name is used repeatedly in a method, you can use a ``__``
+keyword to avoid argument name collisions.
+
+.. code-block:: pycon
+
+    >>> obj.performSelector(
+    ...     SEL("someMethod:"),
+    ...     withObject__1=data1,
+    ...     withObject__2=data2
+    ... )
+
+In this example, the ``withObject`` arguments are duplicated, so the suffixes
+``__1`` and ``__2`` are added to avoid argument name collisions. Since ``__``
+and any part after it are ignored, it is possible to use any suffix starting
+with it.
+
```

Comment:
`performSelector` isn't a great example here, as it can be easily confused with the fact that we're discussing how to access selectors. Going "meta" in the tutorial is a potential source of confusion.

There's also the following paragraph, which contradicts this one; and the need for API documentation in addition to the tutorial.